docs/release-notes/v1.50-v1.59.md

```diff
@@ -116,6 +116,7 @@ Please review the documentation, as these changes may impact your integrations.
     1. **Tabular Data Endpoint Changes**
 
           * ??? abstract "Introduced changes to Tabular Data"
+
```

Comment:
not sure about these lines

Reply:
they are autogenerated by lint tool. Pre-commit fails otherwise 🤷

Reply:
But this have changed the appearance. The expandable sections became empty.